frontend/src/components/pages/RegistrantExperience/RegistrationSteps/ReviewRegistration/ReviewInformation/EditEDLPCard.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useMemo } from "react";
+import { Accordion, Box, Wrap, WrapItem } from "@chakra-ui/react";
+import EditCardFooter from "./EditCardFooter";
+import EditCardHeader from "./EditCardHeader";
+import { CampResponse, CampSession } from "../../../../../../types/CampsTypes";
+import EDLPSelectionControl from "../../AdditionalInfo/QuestionCards/EDLPSelectionControl";
+import EDLPSessionRegistration from "../../AdditionalInfo/edlpSessionRegistration";
+import { EdlpSelections } from "../../../../../../types/RegistrationTypes";
+
+type EditEDLPCardProps = {
+  requireEDLP: boolean | null;
+  setRequireEDLP: (setRequireEDLP: boolean) => void;
+  selectedSessions: CampSession[];
+  camp: CampResponse;
+  edlpSelections: EdlpSelections;
+  setEdlpSelections: React.Dispatch<React.SetStateAction<EdlpSelections>>;
```

Comment:
could we make the setState types a bit more consistent? I believe the typing can be (setEdlpSelections: EdlpSelections) => void;

Reply:
Good catch, I actually changed the setRequireEdlp (everywhere) to be more consistent with the React.Dispatch form everywhere else since that's more accurate.